events/events.go

```diff
@@ -49,3 +49,9 @@ type LookupEvent struct {
 	Key      string
 	Duration time.Duration
 }
+
+// Ready is fired when ringpop has successfully bootstrapped and is ready to receive requests and other method calls.
+type Ready struct{}
```

Comment:
Yeah, I guess that's true, the swim package also has events, but what you have here is good